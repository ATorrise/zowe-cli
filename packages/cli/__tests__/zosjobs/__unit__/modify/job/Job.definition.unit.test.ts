/*
* This program and the accompanying materials are made available under the terms of the
* Eclipse Public License v2.0 which accompanies this distribution, and is available at
* https://www.eclipse.org/legal/epl-v20.html
*
* SPDX-License-Identifier: EPL-2.0
*
* Copyright Contributors to the Zowe Project.
*
*/

// import * as ModifyDefinition from "../../../../../src/zosjobs/modify/Modify.definition";
 
describe("zos-jobs modify job definition", () => {
    it("should not have changed", () => {
        const definition = require("../../../../../src/zosjobs/modify/Modify.definition").ModifyDefinition;
        // const definition = ModifyDefinition.ModifyDefinition;
        expect(definition).toBeDefined();
        delete definition.children;
        expect(definition).toMatchSnapshot();
    });
});
