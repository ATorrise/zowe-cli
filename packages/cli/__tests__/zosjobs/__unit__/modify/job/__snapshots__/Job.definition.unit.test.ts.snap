// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`zos-jobs modify job definition should not have changed 1`] = `
Object {
  "aliases": Array [
    "mdf",
  ],
  "description": "Modify the job class or the hold status of a job.",
  "name": "modify",
  "summary": "Modify a z/OS job",
  "type": "group",
}
`;
